dipy/reconst/fwdti.py

```diff
@@ -0,0 +1,608 @@
+#!/usr/bin/python
+""" Classes and functions for fitting tensors """
+from __future__ import division, print_function, absolute_import
+
+import warnings
+
+import numpy as np
+
+import scipy.optimize as opt
+
+from .base import ReconstModel
+
+from dipy.reconst.dti import (TensorFit, design_matrix, _min_positive_signal,
+                              decompose_tensor, from_lower_triangular,
+                              lower_triangular, apparent_diffusion_coef)
+from dipy.reconst.dki import _positive_evals
+
+from dipy.core.sphere import Sphere
+from .vec_val_sum import vec_val_vect
+from dipy.core.ndindex import ndindex
+
+
+def fwdti_prediction(params, gtab, Diso=3.0e-3):
+    """
+    Predict a signal given the parameters of the free water diffusion tensor
```

Comment:
Description on the first line of the docstring should preferably be just one line.